### PR TITLE
Added cheatsheet for csgo

### PR DESCRIPTION
**Pull request by aiyankovil**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New CSGO Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes
Added cheatsheet of csgo abbreviations

---
https://duck.co/ia/view/csgo_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@aiyankovil Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/csgo_cheat_sheet_mobile.png?Et2ZYoUMBR)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/csgo_cheat_sheet_mobile.png?y4JeoNMhjH)

File: share/goodie/cheat_sheets/json/csgo.json [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+          },{
+           "key":"GL",
+           "val":"Good Luck"
+          },{
+            "key":"HF",
+           "val":"Have Fun"
+          } ],
+        "In-game": [{
+            "key":"ECO",
+            "val":"'saving round' players buy nothing to save money to the next round. (you can buy small pistols like P250 or some nades It depends how much money do you have)"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/csgo.json]: remove the quote
Avoid periods at end of val fields.
Start val fields with a capital letter

Comment (aiyankovil) [inline share/goodie/cheat_sheets/json/csgo.json]: @jophab THE SPECIFIED CHANGES HAS BEEN MADE.THKS FOR SUGGESTING

**Comment by tagawa** [comment]
Hi @aiyankovil. If you're having trouble with these conflicts, probably the easiest thing is to create a new branch for each JSON file (this one and https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3241) and send them as separate new pull requests. If the two JSON files are on separate branches then there shouldn't be problems.

Let me know if you need any help with this.

File: share/goodie/cheat_sheets/json/csgo.json [hunk 2]
```diff
@@ -0,0 +1,263 @@
+{
+    "id": "csgo_cheat_sheet",
+    "name": "CS-GO Abbreviations",
+    "description": "CS-GO Abbreviations",
+    "metadata": {
+        "sourceName": "steamcommunity",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/csgo.json]: @aiyankovil please make a PR that changes this to "Steam Community"

File: share/goodie/cheat_sheets/json/csgo.json [hunk 3]
```diff
@@ -0,0 +1,263 @@
+{
+    "id": "csgo_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/csgo.json]: I would suggest you add some `alias`es like "cs-go", "counterstrike go", "counter-strike go", etc.

**Comment by daxtheduck** [comment]
@aiyankovil this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**